binderhub/tests/test_registry.py

```diff
@@ -196,6 +204,49 @@ async def test_get_token():
     assert token == test_handle["token"]
 
 
+async def test_get_token_with_config_supplied_service():
```

Comment:
It looks like this is a copy of `test_get_token` apart from the arguments to `registry._get_token()`. Can you use `@pytest.mark.parametrize` instead to reduce duplication?